lib/rubocop/cop/style/inline_comment.rb

```diff
@@ -4,13 +4,26 @@
 module RuboCop
   module Cop
     module Style
-      # This cop checks for inline comments.
+      # This cop checks for trailing comments.
+      #
+      # @example
+      #
+      #   # good
+      #   foo.each do |f|
+      #     # Standalone comment
+      #     f.bar
+      #   end
+      #
+      #   # bad
+      #   foo.each do |f|
+      #     f.bar # Trailing comment
+      #   end
       class InlineComment < Cop
-        MSG = 'Avoid inline comments.'.freeze
+        MSG = 'Avoid trailing comments.'.freeze
 
         def investigate(processed_source)
           processed_source.comments.each do |comment|
-            next unless comment.inline?
+            next if comment_line?(processed_source[comment.loc.line - 1])
```

Comment:
Seems there should still be a check whether the comment is inline or not? Where is this `comment_line?` method coming from?